extensions/machine-learning-services/src/packageManagement/sqlRPackageManageProvider.ts

```diff
@@ -93,20 +99,22 @@ export class SqlRPackageManageProvider extends SqlPackageManageProviderBase impl
 		return false;
 	}
 
+	private getPackageLink(packageName: string): string {
+		return `${this._config.rPackagesRepository}/web/packages/${packageName}`;
+	}
+
 	/**
 	 * Returns package overview for given name
 	 * @param packageName Package Name
 	 */
 	protected async fetchPackage(packageName: string): Promise<nbExtensionApis.IPackageOverview> {
 		let packagePreview: nbExtensionApis.IPackageOverview = {
 			name: packageName,
-			versions: [],
+			versions: [constants.latestVersion],
```

Comment:
You sure you want to always fetch the latest? That might break people if an update is released before we can test it. A better idea might be to just specify a known good version and have that be a config value - then you can always document the way for users to update that value manually if they really need the latest version.

Reply:
I like the idea of being able to specify your own version, but I think we should flip it around: default to latest stable (maybe within a major version? idk), then document the way for users to update that value to stick to a specific version.  Two reasons are 1) we should trust package stewards with their own quality, and 2) we don't want to get in the business of pushing an update (even a small one with just an incrementation in a config) each time a package updates.

Reply:
don't have any choice right now. the library I'm using doesn't get version

---
In reply to: [369283258](https://github.com/microsoft/azuredatastudio/pull/8920#discussion_r369283258) [](ancestors = 369283258)